src/external_coding_agent.py

```diff
@@ -41,10 +41,75 @@ def __init__(self, config: Config):
         self.config = config
         debug_log(f"Initialized ExternalCodingAgent")
     
-    def generate_fixes(self, vuln_uuid: str, remediation_id: str, vuln_title: str) -> bool:
+    def assemble_issue_body(self, vulnerability_details: dict) -> str:
+        """
+        Assembles a GitHub Issue body from vulnerability details.
+        
+        Args:
+            vulnerability_details: Dictionary containing vulnerability information
+            
+        Returns:
+            str: Formatted GitHub Issue body for the vulnerability
+        """
+        # Extract key details with safe fallbacks
+        vuln_title = vulnerability_details.get('vulnerabilityTitle', 'Unknown Vulnerability')
+        vuln_uuid = vulnerability_details.get('vulnerabilityUuid', 'Unknown UUID')
+        vuln_rule = vulnerability_details.get('vulnerabilityRuleName', 'Unknown Rule')
+        vuln_severity = vulnerability_details.get('vulnerabilitySeverity', 'Unknown Severity')
+        vuln_status = vulnerability_details.get('vulnerabilityStatus', 'Unknown Status')
+        vuln_overview = vulnerability_details.get('vulnerabilityOverviewStory', 'No overview available')
+        vuln_events = vulnerability_details.get('vulnerabilityEventsSummary', 'No event details available')
+        vuln_http_details = vulnerability_details.get('vulnerabilityHttpRequestDetails', 'No HTTP request details available')
+        
+        # Assemble the issue body
+        issue_body = f"""
+# Contrast AI SmartFix Issue Report
+
+This PR addresses a vulnerability identified by the Contrast Security platform (ID: [{vuln_uuid}](https://{self.config.CONTRAST_HOST}/Contrast/static/ng/index.html#/{self.config.CONTRAST_ORG_ID}/applications/{self.config.CONTRAST_APP_ID}/vulns/{vuln_uuid})).
+        
+# Security Vulnerability: {vuln_title}
```

Comment:
after the demo, I feel we should pull this from a template file and make it look a little fancier 😄